crypto/mem_dbg.c

```diff
@@ -443,7 +443,8 @@ void CRYPTO_mem_debug_realloc(void *addr1, void *addr2, size_t num,
 }
 
 typedef struct mem_leak_st {
-    BIO *bio;
+    int (*print_cb) (const char *str, size_t len, void *u);
```

Comment:
I modeled this after `ERR_print_errors_cb`...  And quite frankly, since this is getting into the public API, I'd prefer to keep the length parameter, just in case we *do* get to need it some day.  It doesn't hurt.